src/EnergyPlus/WeatherManager.hh

```diff
@@ -198,6 +198,10 @@ namespace WeatherManager {
         bool ActualWeather;              // true when using actual weather data
         int RawSimDays;                  // number of basic sim days.
         bool firstHrInterpUseHr1;        // true when using Hour 1 for first hour interpolations; false to use Hour 24
+        Real64 maxCoolingOATSizing;      // max outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
+        Real64 maxCoolingOADPSizing;     // max outdoor dew point for DesignDay or RunPeriodDesign type weather
+        Real64 minHeatingOATSizing;      // min outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
+        Real64 minHeatingOADPSizing;     // min outdoor dew point for DesignDay or RunPeriodDesign type weather
```

Comment:
@Myoldmopar if you are making changes, you could correct the comments on DP. 

    outdoor dew point at max outdoor dry-bulb for DesignDay or RunPeriodDesign type weather
    outdoor dew point at min outdoor dry-bulb for DesignDay or RunPeriodDesign type weather